src/NuGet.Clients/NuGet.CommandLine/Common/ConsoleProjectContext.cs

```diff
@@ -65,6 +66,6 @@ public virtual ProjectManagement.FileConflictAction ResolveFileConflict(string m
 
         public NuGetActionType ActionType { get; set; }
 
-        public INuGetTelemetryService TelemetryService { get; set; }
+        public KeyValuePair<string, Guid> OperationId { get; set; }
```

Comment:
for that matter why do we need a kvp here?

Reply:
becasue telemetryService accpet a kvp